modules/move-etcd-different-disk.adoc

```diff
@@ -15,6 +15,11 @@ The Machine Config Operator (MCO) is responsible for mounting a secondary disk f
 This procedure does not move parts of the root file system, such as `/var/`, to another disk or partition on an installed node.
 ====
 
+.Limitations
+
+* When the new disk is attached to the cluster, the etcd database is part of the root mount. It is not part of the secondary disk or the intended disk when the master node is recreated. As a result, the master node will not create a separate `/var/lib/etcd` mount.
+* After the nodes are on a separate disk, you must update the `etcd-mc.yml` file to replace the instances of `_<new_disk_name>_` with the device name.
```

Comment:
This is already mentioned in each applicable step of the procedure. Is mentioning it here redundant, or am I misunderstanding the point of including it here?

Reply:
We are not mentioning that when a new master node is created via machineset or via iso install the secondary device would not be formatted, as a .mount unit would exist for /var/lib/etcd the system will fail to boot.
Hence we need to know mention limitations, also we are changing the documentation to improve the steps of migrating etcd to secondary device. The current method has lot of issues.